src/logical/backend.rs

```diff
@@ -60,6 +63,13 @@ impl Backend for LogicalBackend {
             return Err(RvError::ErrRequestNotReady);
         }
 
+        match req.operation {
+            Operation::Renew | Operation::Revoke => {
+                return self.handle_revoke_renew(req);
```

Comment:
What is the consideration on this? Why don't separate revoke and renew into 2 functions since you need to do the `match` work again in the following `handle_auth_renew` function.

Reply:
> What is the consideration on this? Why don't separate revoke and renew into 2 functions since you need to do the `match` work again in the following `handle_revoke_renew` function.

The reusable code for 'revoke' and 'renew' has been extracted into the handle_revoke_renew function.